lib/scrape-worker.js

```diff
@@ -0,0 +1,10 @@
+const Request = require('./Request');
```

Comment:
very minor thing - generally you want your functions to be camelcase. Starting with an uppercase letter should just be for classes and react components